tests/testModuleImporterLib.cc

```diff
@@ -26,10 +26,14 @@
 
 namespace py = pybind11;
 
+namespace {
```

Comment:
not that I mind this being here at all, I am just curious why it is important in a test function?

Reply:
It probably doesn't matter, but it is a library so I felt it safest.